drizzlepac/haputils/poller_utils.py

```diff
@@ -491,16 +491,19 @@ def parse_mvm_tree(det_tree, all_mvm_exposures, log_level):
                 prod_info = (filename[0] + " " + filetype).lower()
                 #
                 # mvm prod_info = 'skycell_p1234_x01y01 wfc3 uvis f200lp all 2009 1 drz'
+                
+                # prod_list is 0: proposal_id, 1:observation_id, 2:instrument, 3:detector, 
```

Comment:
It looks like the old comment here is not correct, and the new comment is using the SVM terminology, so it is also not correct.  The problem here is the MVM poller historically has not been parsed to collect quite the same information as the SVM poller, and the comment here was not updated long ago to contain the correct information. 

Line 495 (prod_list) should reflect the actual tokenized values from LIne 493 (prod_info).  Please fix the Line 495 comment as the values are skycell, instrument, detector, filters, exposure_layer, year_layer, old_or_new, and filetype (CTE corrected or not).